packages/functions/src/load-balancing/scheduledMoveGame.ts

```diff
@@ -0,0 +1,36 @@
+import * as functions from 'firebase-functions';
+import * as admin from "firebase-admin";
+import {FirebaseCollection} from '@pipeline/common';
+import {getDatabase} from "../utils/rtdb";
+import {Game} from "../models/Game";
+import {moveGameFromRTDBToFirestore} from "./utils";
+const db = admin.firestore();
+const logger = functions.logger;
+
+const DAY_IN_MILLIS = 24 * 60 * 60 * 1000;
+
+const moveGamesJob = async () => {
+  const lastActiveGamesDate = new Date(Date.now() - DAY_IN_MILLIS);
+
+  const oldActiveGamesQuery = db.collection(FirebaseCollection.Games)
+      .where(`lastPlayerDisconnectedAt`, '<=', lastActiveGamesDate);
+
+  const oldActiveGamesSnap = await oldActiveGamesQuery.get();
```

Comment:
todo to add pagination for reliability?

Reply:
The job has 2GB of memory and runs every 60 minutes. It's very unlikely to have hundreds/thousands of game in that state in only 60 min